src/main/msp/msp.c

```diff
@@ -1833,7 +1829,7 @@ case MSP_NAME:
 
         // API 1.41 - Add multi-gyro indicator, selected gyro, and support for separate gyro 1 & 2 alignment
         sbufWriteU8(dst, getGyroDetectionFlags());
-#ifdef USE_MULTI_GYRO
+#if GYRO_COUNT > 1
         sbufWriteU8(dst, gyroConfig()->gyro_to_use);
```

Comment:
GYRO_CONFIG_USE_GYRO_BOTH == 2 is problematic here. 
Probably best way is to refactor `gyro_to_use` as bitmask and use bit tricks to derive used gyros where necessary

Or accept that 2 = ALL, 3 = third gyro